cli-plugins/apps/tutorials/get-workload-status.hbs.md

```diff
@@ -0,0 +1,635 @@
+        managedFields:
+        ...
+        ...
+        manager: cartographer
+        operation: Update
+        time: "2022-05-17T22:06:52Z"
+    name: tanzu-java-web-app
+    namespace: default
+    resourceVersion: "6071972"
+    uid: 7fbcd40d-4eb3-41dc-a1db-657b64148708
+    spec:
+        source:
+            git:
+                ref:
+                  tag: tap-1.3
+                url: https://github.com/vmware-tanzu/application-accelerator-samples
+            subPath: tanzu-java-web-app
+    ...
+    ...
+    ```
+
+- json
+
+    ```bash
+    tanzu apps workload list --output json
+    ```
+
+    ```json
+    [
+        {
+            "kind": "Workload",
+            "apiVersion": "carto.run/v1alpha1",
+            "metadata": {
+                "name": "tanzu-java-web-app2",
+                "namespace": "default",
+                "uid": "7fbcd40d-4eb3-41dc-a1db-657b64148708",
+                "resourceVersion": "6071972",
+                "generation": 1,
+                "creationTimestamp": "2022-05-17T22:06:49Z",
+                "labels": {
+                    "app.kubernetes.io/part-of": "tanzu-java-web-app",
+                    "apps.tanzu.vmware.com/workload-type": "web"
+                },
+            ...
+            }
+        ...
+        },
+        {
+            "kind": "Workload",
+            "apiVersion": "carto.run/v1alpha1",
+            "metadata": {
+                "name": "tanzu-java-web-app",
+                "namespace": "default",
+                "uid": "7fbcd40d-4eb3-41dc-a1db-657b64148708",
+                "resourceVersion": "6071972",
+                "generation": 1,
+                "creationTimestamp": "2022-05-17T22:06:49Z",
+                "labels": {
+                    "app.kubernetes.io/part-of": "tanzu-java-web-app",
+                    "apps.tanzu.vmware.com/workload-type": "web"
+                },
+            ...
+            }
+        ...
+        },
+    ...
+    ...
+    ]
+    ```
+
+## Workload get
+
+The `tanzu apps workload get` command retrieves information and status about a workload. Workloads
+can be filtered with the `--namespace` flag, which specifies the namespace where the workload
+is deployed.
+
+There are multiple sections in the workload get command output. The following data is displayed:
+
+- Name of the workload and its status.
+- Displays source information of workload.
+- If the workload was matched with a supply chain, the information of its name and the status is displayed.
+- Information and status of the individual steps that is defined in the supply chain for the workload.
+- Any issue with the workload: the name and corresponding message.
+- Workload related resource information and status like services claims, related pods, knative services.
+
+At the very end of the command output, a hint to follow up commands is also displayed.
+
+>**Note** the `Supply Chain` and `Delivery` sections are included in the command output depending on
```

Comment:
You don't need to add multiple ">" - I've removed them in the example below:

>**Note** the `Supply Chain` and `Delivery` sections are included in the command output depending on whether those resources are present on the target cluster (e.g. If the target includes only build components, there would be no `Delivery` resources available and therefore the `Delivery` section would not be included in the command output.).

Reply:
ok, removed additional characters